subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1585,6 +1589,10 @@ static uint8_t vs_cmd_put(uint8_t const *const cmd, uint8_t *const raw_event_out
 		*param_length_out += sizeof(sdc_hci_cmd_vs_get_next_conn_event_counter_return_t);
 		return sdc_hci_cmd_vs_get_next_conn_event_counter((void *)cmd_params,
 							    (void *)event_out_params);
+#endif
+#if defined(CONFIG_BT_CTLR_SDC_PAWR_ADV)
```

Comment:
Should there be an `&& defined(CONFIG_BT_CENTRAL)` here? There is no use for the command if the application only uses pawr and never runs an initiator.

Reply:
Are you thinking about the use-case where we would support periodic advertising with responses + central links, but not scanning?

I don't think it really matters, given the command is just one line

Reply:
I was thinking about supporting just pawr and no connections.